### PR TITLE
feat: setup-kalix-action initial release

### PR DESCRIPTION
**Pull request by aludwiko**

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+* Follow 
+* Create a [new release](https://github.com/lightbend/kalix-action/releases/new) with a new tag and the title equal to the tag
+*   Choose the version judiciously, as making the version a minor number bump will allow existing users to be upgraded seamlessly. A major number bump will force them to update their action. Follow this [instruction](https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-release-management-for-actions)
+* Update [README.md](README.md) to show the new tag
+* Update [Kalix Docs](https://github.com/lightbend/kalix-docs/blob/fdfd537fdc5ca9eb2a828b947b99a2ab52dce163/docs/modules/operations/pages/integrate-cicd-github-actions.adoc) to
+  the major version
+* Update [kalix-cicd-github-actions/README.md](https://github.com/lightbend/kalix-cicd-github-actions) to show the new tag
```
Comment (ennru) [inline RELEASING.md:7]: Copy-paste from `kalix-action`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* run `ncc build index.js`
+* commit sources and `dist` folder
+
+## Contributing
+
+We welcome all contributions! [Pull requests](https://github.com/lightbend/setup-kalix-action/pulls) are the preferred way to share your contributions. For major changes, please open [an issue](https://github.com/lightbend/setup-kalix-action/issues) first to discuss what you would like to change.
+
+## Support
+
+This project is an [incubator](https://developer.lightbend.com/docs/introduction/getting-help/support-terminology.html)
```
Comment (ennru) [inline README.md]:
```suggestion
This project is an [incubator](https://doc.akka.io/docs/akka-dependencies/current/support-terminology.html#incubating)
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# Kalix CLI for GitHub Actions
+
+This [Java Script action](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action) installs and configures [Kalix CLI](https://docs.kalix.io/kalix/index.html) tool. After this action is run, you can use the `kalix` command in your workflow.
```
Comment (ennru) [inline README.md]:
```suggestion
This [JavaScript action](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action) for GitHub workflows installs and configures the [Kalix CLI](https://docs.kalix.io/kalix/index.html) tool. After running this action, the `kalix` command is available in the workflow. 
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+# Kalix CLI for GitHub Actions
+
+This [Java Script action](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action) installs and configures [Kalix CLI](https://docs.kalix.io/kalix/index.html) tool. After this action is run, you can use the `kalix` command in your workflow. 
+
+## Usage
+
+The action takes two required variables to authenticate and set the project you want to work on:
```
Comment (ennru) [inline README.md]:
```suggestion
The action takes two required parameters to authenticate and set the Kalix project ID:
```

**Review by ennru — APPROVED** [review]
LGTM.

File: RELEASING.md (hunk 1 above)
Comment (aludwiko) [reply RELEASING.md:7]: yes, but it's pretty valid

**Pull request merged**